h/static/scripts/group-forms/utils/api.ts

```diff
@@ -45,15 +45,24 @@ export class APIError extends Error {
 /* Make an API call and return the parsed JSON body or throw APIError. */
 export async function callAPI(
   url: string,
-  method: string = 'GET',
-  json: object | undefined,
+  {
+    method = 'GET',
+    json = null,
+    headers = {},
+  }: {
+    method?: string;
+    json?: object | null;
+    headers?: any;
+  } = {},
 ): Promise<object> {
+  headers['Content-Type'] = 'application/json; charset=UTF-8';
```

Comment:
This mutates the input object, which the caller might not expect. A way to avoid this is to use object spread syntax to do a shallow copy + update:

```js
const options = {
  headers: {
     ...headers,
    'Content-Type': '...',
  },
  ...
}
```

Reply:
Done: https://github.com/hypothesis/h/pull/8864/commits/6f2f2236bb51360b1be8cd19f41eb4a76a636f10